src/Main.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import HornedBeasts from './HornedBeasts.js';
+
+
+class Main extends React.Component{
+  render(){
+    return(
+      <main>
```

Comment:
- Awesome work!! Lovin' the animals 💯 

- on line 8, we see `<main>`
-  Instead we see that fragments `<>`, or a `<div>`s are used commonly for containers, although now I think now the consensus is that fragments are are ideal